expression/builtin_time.go

```diff
@@ -2039,69 +2041,102 @@ type timestampFunctionClass struct {
 	baseFunctionClass
 }
 
+func (c *timestampFunctionClass) getDefaultFsp4Type(tp *types.FieldType) int {
+	if tp.Tp == mysql.TypeDatetime || tp.Tp == mysql.TypeDate || tp.Tp == mysql.TypeDuration || tp.Tp == mysql.TypeTimestamp || tp.Tp == mysql.TypeNewDate {
```

Comment:
this line is too long, we can split it into two lines

Reply:
ok